### PR TITLE
Dependency upgrade [RA-2244]

### PR DESCRIPTION
**Pull request by gsanta**

### CODE REVIEW

File: .tool-versions [hunk 1]
```diff
@@ -1,2 +1,2 @@
-ruby 3.2.1
-nodejs 16.17.1
+ruby 3.2.2
+nodejs 18.16.0
```
Comment (moczolaszlo) [inline .tool-versions:2]: Is this could be the current LTS? (20.10.0), Is WFE works with that?

File: spec/integration/cypress/plugins/index.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ const cucumber = require("cypress-cucumber-preprocessor").default
 /**
  * @type {Cypress.PluginConfig}
  */
-module.exports = (on, config) => {
+module.exports = (on) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
```
Comment (moczolaszlo) [inline spec/integration/cypress/plugins/index.js:21]:
```suggestion
```

**Review by moczolaszlo — COMMENTED** [review]
I only have one question. :)

File: .tool-versions (hunk 1 above)
Comment (gsanta) [reply .tool-versions:2]: I will try!
Wanted to make it the same version as the website but if it works with the latest why not.

File: spec/integration/cypress/plugins/index.js (hunk 2 above)
Comment (gsanta) [reply spec/integration/cypress/plugins/index.js:21]: I removed this because with the updated linter it threw an `unused variable` error

Comment (moczolaszlo) [reply spec/integration/cypress/plugins/index.js:21]: That ok, just remove the comment too 😄

Comment (gsanta) [reply spec/integration/cypress/plugins/index.js]: Ah I misunderstood :) good catch

**Review by moczolaszlo — APPROVED** [review]

**Review by moczolaszlo — APPROVED** [review]

**Pull request merged**